test/integration/testdata/rbac-beta.yaml.tmpl

```diff
@@ -12,7 +12,10 @@ rules:
   resources: ["istiokinds"]
   verbs: ["get", "list", "watch"]
 - apiGroups: ["apiextensions.k8s.io"]
-  resources: ["customresourcedefinitions"]
+  resources: ["*"]
```

Comment:
Well, this is the future while istio/istio one is the past :) I'd keep this as precise as possible but since it's just too painful to coordinate with istio/istio, leave the released ones more open.